pgdrive/envs/multi_agent_pgdrive.py

```diff
@@ -218,60 +220,37 @@ def _force_respawn(self, agent_name):
         This function can force a given vehicle to respawn!
         """
         self._agent_manager.finish(agent_name)
+        if agent_name == self._agent_manager.object_to_agent(self.current_track_vehicle.name):
+            self.chase_another_v()
```

Comment:
Why this function is called twice? I think the _after_vehicle_done function already solve this issue!

Reply:
Solved, and fix the camera bug